test-packages/release/src/publish.ts

```diff
@@ -12,6 +13,14 @@ function tagFor(pkgName: string, entry: { newVersion: string }): string {
   return `v${entry.newVersion}-${pkgName.replace(/^@embroider\//, '')}`;
 }
 
+function info(message: string) {
+  process.stdout.write(`\n ℹ️  ${message}`);
```

Comment:
Is the extra space (two instead of one) after :info: intentional? The message does not align properly currently 😅 

```
 ℹ️  The tag, v3.1.1-compat, has already been pushed up for @embroider/compat
 ℹ️  --dry-run active. Skipping `git push --tags`
 ℹ️  A release with the name 'v3.1.1-compat' already exists
 ℹ️  @embroider/compat has already been publish @ version 3.1.1
 🎉 --dryrun active. Would have successfully published release! 🎉
```